### PR TITLE
Update Benefits section on the Careers page

### PR DESCRIPTION
**Pull request by infraredgirl**
- Mention that we have location independent salaries
- Remove mentions of performance bonus
- Group US-specific benefits together
- Correct the value for wellness budget

### CODE REVIEW

File: pages/careers/_human-friendly.html
Comment (josh-padnick) [inline pages/careers/_human-friendly.html:22]:
```suggestion
      <li><strong>Location Independent, Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and uses a multiplier to produce a result that's above market for that title. Our salaries are location independent.</li>
```

File: pages/careers/_human-friendly.html
Comment (josh-padnick) [inline pages/careers/_human-friendly.html:25]:
```suggestion
      <li><strong>Personal Budget.</strong> We'll give you a personal budget of $1,000 USD per month to spend on your workspace (e.g., a co-working space), health (e.g., gym, yoga), time (e.g., babysitter), and/or learning (e.g., books, courses).</li>
```

File: pages/careers/_human-friendly.html
Comment (josh-padnick) [inline pages/careers/_human-friendly.html:31]: We contribute to everyone's pension; it's just that it's optional in the USA! Would you mind moving that above the US-specific label?

File: pages/careers/_human-friendly.html
Comment (josh-padnick) [inline pages/careers/_human-friendly.html:29]: We actually do pay for "premium" health coverage in other countries per https://www.notion.so/gruntwork/Benefits-by-Country-644686d34b4641bf8853754fc49749ff, so could we move this to the "global" section as well (versus the US-specific section)?

**Review by josh-padnick — CHANGES_REQUESTED** [review]
Thanks, Ana! This looks solid overall; I just left a few comments around accuracy.

File: pages/careers/_human-friendly.html
Comment (infraredgirl) [reply pages/careers/_human-friendly.html]: Ah, that's a good point, I forgot about the extra health coverage! Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/0f144aaa4f9c31160c31ca073c20c20a9e483fa4 and I added a sentence to emphasize the extra aspect of this for non-US countries.

File: pages/careers/_human-friendly.html
Comment (infraredgirl) [reply pages/careers/_human-friendly.html]: Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/0f144aaa4f9c31160c31ca073c20c20a9e483fa4.

File: pages/careers/_human-friendly.html
Comment (infraredgirl) [reply pages/careers/_human-friendly.html]: Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/0f144aaa4f9c31160c31ca073c20c20a9e483fa4.

File: pages/careers/_human-friendly.html
Comment (infraredgirl) [reply pages/careers/_human-friendly.html:31]: I see what you're saying, however to me this reads as Gruntwork contributes to a private pension plan (which would be on top of statutory pension), which is not the case for any of the non-US countries if I'm reading the Benefits page right. Speaking for the EU at least, the statutory pension is guaranteed by law, so it's not customary to call that out in a job post. I propose that we leave this item under US-specific section, but remove mention of "pension" and leave only 401K. That way the US candidates now that they get 401K, and non-US candidates don't get the false impression that they will get private pension contributions as a benefit. What do you think?

Comment (josh-padnick) [reply pages/careers/_human-friendly.html:31]: Ah, that sounds perfect, and thank you for helping me understand convention around the EU!

**Review by josh-padnick — APPROVED** [review]
Thanks, Ana. I think we're good here! I've approved though I believe there are other changes and you may need one more approval before merging.

File: pages/careers/_human-friendly.html
Comment (infraredgirl) [reply pages/careers/_human-friendly.html]: Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/593ef439f8119da4388845212a1687d1b0e24177.

**Review by josh-padnick — APPROVED** [review]

**Pull request merged**
